lib/challenge.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
+let milli = 1000000;
+let sumTen = 0;
+let sumTwo = 0;
+
+String.prototype.reverse = function() {
+	return this.split('').reverse().join('');
+};
```

Comment:
Rewriting the prototype method? 
'It’s a bold strategy Cotton. Let’s see if it pays off for him.'